pkg/controllers/organization/organization_controller.go

```diff
@@ -75,3 +84,47 @@ func (r *NamespaceReconciler) reconcileNamespace(ctx context.Context, org *green
 	}
 	return nil
 }
+
+func (r *OrganizationReconciler) reconcileAdminTeam(ctx context.Context, org *greenhousesapv1alpha1.Organization) error {
+	orgAdminTeamName := org.Name + "-admin"
+	namespace := org.Name
+
+	orgAdminTeamExists := true
+	var orgAdminTeam = new(greenhousesapv1alpha1.Team)
+	if err := r.Get(ctx, types.NamespacedName{Name: orgAdminTeamName, Namespace: namespace}, orgAdminTeam); err != nil {
+		if apierrors.IsNotFound(err) {
+			orgAdminTeamExists = false
+		} else {
+			return err
+		}
+	}
+
+	mappedOrgAdminIDPGroup := org.Spec.MappedOrgAdminIDPGroup
+
+	if orgAdminTeamExists && orgAdminTeam.Spec.MappedIDPGroup == mappedOrgAdminIDPGroup {
+		// Nothing changed.
+		return nil
+	}
```

Comment:
This is not necessary when using [clientutil.CreateOrPatch(..)](https://github.com/cloudoperators/greenhouse/blob/77fb415b8df8fa7ae636fb1e2b19a710dc81f635/pkg/clientutil/patch.go#L69-L71). The helper will do the create if the object does not exist. 
Also it will check and skip if the existing and the patched object are equal.

Reply:
Fixed.